<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Micah's Libre Calendar Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="monthList">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Calendar">
    <property name="title">
     <string>&amp;Calendar</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="action_Print_Setup"/>
    <addaction name="action_Quit"/>
    <addaction name="action_Generate_Calendar"/>
    <addaction name="action_Options"/>
   </widget>
   <addaction name="menu_Calendar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Start a new calendar project</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="toolTip">
    <string>Load a calendar project</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save your work</string>
   </property>
  </action>
  <action name="action_Print_Setup">
   <property name="text">
    <string>&amp;Print Setup</string>
   </property>
   <property name="toolTip">
    <string>Set up the printing parameters</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Generate_Calendar">
   <property name="text">
    <string>&amp;Generate Calendar</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
